scripts/tables.sql

```diff
@@ -177,4 +175,4 @@ INSERT INTO migrations (name)
 SELECT '202307111517030_add_subscription_invitations_table'
```

Comment:
Modify last migration reference to yours

Reply:
Done